property-panel-ux/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
should we generate these examples with a ts config that fixes the issues with vs-code imports?